_examples/mini-habr-with-subscriptions/go.mod

```
@@ -0,0 +1,44 @@
+module github.com/nabishec/ozon_habr_api
```

Can you change the module to match the directory structure? 
```suggestion
modulemodule github.com/gqlgen/_examples/mini-habr-with-subscriptions
```